src/main/resources/travis/travis.wdl

```diff
@@ -0,0 +1,31 @@
+task centaur {
+    String cromwell_branch
+    File conf
+    File pem
+    File cromwell_jar
+
+    command<<<
+        mkdir -p /cromwell_root/tmp/ivy2
+        export SBT_OPTS=-Dsbt.ivy.home=/cromwell_root/tmp/.ivy2
+        git clone https://github.com/broadinstitute/centaur.git
+        cd centaur
+        git checkout jg_allow_cromwell_jar
```

Comment:
This needs to be changed, but waiting on other slackasses to review the centaur PR (just officially assigned @cjllanwarne and @Horneth to that)